rust/lance-linalg/build.rs

```diff
@@ -64,15 +63,20 @@ fn main() {
             // We create a special cfg so that we can detect we have in fact
             // generated the AVX512 version of the f16 kernels.
             println!("cargo:rustc-cfg=kernel_suppport=\"avx512\"");
+            return Ok(());
```

Comment:
Why do we return here? We want to build both AVX-512 *and* AVX2 (for runtime dispatch)

Reply:
Did not realize that.  Good catch.

Reply:
The return has been removed.